### DISCUSSION TITLE
Ublocks  blocks avatar images on some forums

### DISCUSSION THREAD

**Discussion by s1raz1** [uBlockOrigin/uAssets#17137] [General]
For a while now ublock blocks avatar images on  the  site 1337x  while also making it so i cant change pages if there are more than 1 page of comments.   I dont know if this is a bug or something in my options, i dont really fiddle with the settings so i have  the default ones.

**Comment by iam-py-test** [comment]
This is a filterlist issue, and should be reported to https://github.com/uBlockOrigin/uAssets/issues

**Comment by mapx-** [comment]
@s1raz1 continue here: https://github.com/uBlockOrigin/uAssets/issues/17138
